src/UiTPAS/Event/EventProcessManager.php

```diff
@@ -82,21 +74,51 @@ private function handleEventCardSystemsUpdated(EventCardSystemsUpdated $eventCar
 
         $this->logger->info('Handling updated card systems message for event ' . $eventId);
 
-        $uitpasLabels = $this->uitpasLabelsRepository->loadAll();
+        $uitPasLabels = $this->uitpasLabelsRepository->loadAll();
+
+        $expectedUitPasLabelsForEvent = array_map(
```

Comment:
I was a bit confused when I saw `$expected` in a non-test context. Made me wonder if we should rethink some of the variables used here.
e.g.:
`$uitPasLabels` -> `$availableUitpasLabels`
`$expectedUitPasLabelsForEvent` -> `$requestedUitPasLabelsForEvent` (not convinced yet about this one)

Reply:
I see your point. My original thought process was that it's a list of labels that should be on the event based on the card systems, but some may already be on the event from before so `labelsToAdd` or something like that didn't seem correct. Since we kind of create a diff and dispatch various commands to get to that state, "expected" seemed better because it described the outcome rather then the action. But it can be confusing because of the convention to use `expected` in tests.

Would prefer something along the lines of `$applicableUitPasLabelsForEvent` over `$requestedUitPasLabelsForEvent` though since technically nothing is being requested (a domain message was sent from UiTPAS notifying is of the update of card systems on the event, but UiTPAS itself does not care about the labels AFAIK).

If you agree feel free to pick this up whenever you have some time, or I can pick it up tomorrow afternoon or  Wednesday.